README.md

```diff
@@ -42,7 +42,7 @@ Default: `'JST'`
 The namespace in which the precompiled templates will be assigned.  *Use dot notation (e.g. App.Templates) for nested namespaces or false for no namespace wrapping.*  When false with `amd` or `commonjs` option set `true`, templates will be returned directly from the AMD/CommonJS wrapper.
 
 Example:
-```js
+```javascript
```

Comment:
I honestly don't see how they get in the way. Nor how `js` was broken.

I just tried being consistent. And Markdown needs two trailing spaces for line breaks as you know.

Formatting looks bad without the line breaks. I'd advice you to stick with the current scheme. If you really don't like the line breaks we should all agree to another solution. Maybe using `*` (ul).